Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -72,6 +72,13 @@ def norm(self, **kwargs):
     def set_norm(self, norm=None):
         '''Sets the norm of the operator to a custom value.
         '''
+        
+        if norm is not None and isinstance(norm, Number) is False:

```

Comment:
Change this to a nested check. First check it is not None, then a number, then positive

Reply:
Done